rabbitmq-server-4.0.yaml

```diff
@@ -150,14 +153,54 @@ subpackages:
 
 update:
   enabled: true
+  ignore-regex-patterns:
+    - .*alpha.*
+    - .*beta.*
+    - .*rc.*
   github:
     identifier: rabbitmq/rabbitmq-server
     strip-prefix: v
     tag-filter-prefix: v4.0.
 
 test:
+  environment:
+    contents:
+      packages:
+        - curl
+        - shadow
+        - sudo-rs
   pipeline:
-    # AUTOGENERATED
-    - runs: |
-        rabbitmq-plugins version
-        rabbitmq-plugins --version
+    - uses: test/daemon-check-output
+      with:
+        setup: |
+          groupadd -r rabbitmq
```

Comment:
Can we consider using declarative accounts block to achieve this?

Reply:
Oh, TIL that we can do it melange too now, thanks will do it

Reply:
We can't do accounts block here, it leads to auth errors and failures with rabbitmq startup as we can't leverage /sbin/nologin then in testing env